### PR TITLE
Table Extracton

### PR DESCRIPTION
**Pull request by kbirk**

### CODE REVIEW

File: packages/server/src/test/java/software/uncharted/terarium/hmiserver/TerariumApplicationTests.java [hunk 1]
```diff
@@ -16,7 +15,6 @@
 
 @AutoConfigureMockMvc
 @SpringBootTest
-@Transactional
```
Comment (dvince2) [inline packages/server/src/test/java/software/uncharted/terarium/hmiserver/TerariumApplicationTests.java:19]: 👍

**Review by dvince2 — APPROVED** [review]

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/ClientEventType.java [hunk 2]
```diff
@@ -20,9 +20,12 @@ public enum ClientEventType {
 	TASK_GOLLM_ENRICH_AMR,
 	TASK_MIRA_AMR_TO_MMT,
 	TASK_MIRA_GENERATE_MODEL_LATEX,
-	TASK_ENRICH_AMR,
+	TASK_ENRICH_AMR, // deprecated use TASK_GOLLM_ENRICH_AMR
```
Comment (YohannParis) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/ClientEventType.java:23]: @dgauldie I created an issue for it #4909

File: packages/table_extraction/table-extraction-version.txt
Comment (YohannParis) [inline packages/table_extraction/table-extraction-version.txt:1]: Why do we need this? Isn't table extraction part of the _Terarium_ repo?

**Review by YohannParis — APPROVED** [review]

File: packages/table_extraction/table-extraction-version.txt
Comment (kbirk) [reply packages/table_extraction/table-extraction-version.txt:1]: Table extraction uses this repo: https://github.com/DARPA-ASKEM/document_intelligence/tree/main/document_intelligence/table_extractor

**Pull request merged**